es6-modules/src/components/todo-list.js

```diff
@@ -0,0 +1,53 @@
+// const { createElement } = require('../lib');
+import createElement from '../lib';
```

Comment:
`createElement` should be a "named export" so it should use curly brackets for the import.
```JAVASCRIPT
import { createElement } from '../lib';
```